### PR TITLE
Docker Desktop 4.21.1 release notes

### PR DESCRIPTION
**Pull request by lorenrh**
### Proposed changes

Docker Desktop 4.21.1 release notes

### CODE REVIEW

File: desktop/release-notes.md [hunk 1]
```diff
@@ -28,7 +28,22 @@ For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/re
...
 {% include release-date.html date="2023-06-29" %}
 
-{% include desktop-install.md all=true version="4.21.0" build_path="/" %}
+{% include desktop-install.md all=true version="4.21.0" build_path="/113844/" %}
+
+### Upgrades
+- [Compose v2.19.1](https://github.com/docker/compose/releases/tag/v2.19.1)
+
+### Bug fixes and enhancements
+
```
Comment (crazy-max) [inline desktop/release-notes.md:37]:
```suggestion

#### For all platforms

- Fixed connection leak for Docker contexts using SSH ([docker/for-mac#6834](https://github.com/docker/for-mac/issues/6834) and [docker/for-win#13564](https://github.com/docker/for-win/issues/13564))
```

cc @laurazard

Comment (laurazard) [reply desktop/release-notes.md:37]: LGTM, thanks @crazy-max. Cc @lorenrh

File: desktop/release-notes.md [hunk 2]
```diff
@@ -28,7 +28,22 @@ For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/re
 
 {% include release-date.html date="2023-06-29" %}
```
Comment (usha-mandya) [inline desktop/release-notes.md:29]: @lorenrh I guess we should update the version number and release date?

Comment (lorenrh) [reply desktop/release-notes.md]: Oh yes! my bad!

File: desktop/release-notes.md (hunk 1 above)
Comment (lorenrh) [reply desktop/release-notes.md:37]: Nice! thanks!

**Comment by dvdksn** [comment]
Can you try rebasing to see if it fixes the build error?

File: desktop/release-notes.md [hunk 3]
```diff
@@ -26,9 +26,28 @@ For frequently asked questions about Docker Desktop releases, see [FAQs](faqs/re
 
 ## 4.21.0
```
Comment (usha-mandya) [inline desktop/release-notes.md:27]:
```suggestion
## 4.21.1
```

**Review by usha-mandya — APPROVED** [review]

**Pull request merged**